cirq-core/cirq/ops/parallel_gate_test.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+import sympy
+import cirq
+
+
+@pytest.mark.parametrize(
+    'gate, num_copies, qubits',
+    (
```

Comment:
turbo nit: I would suggest using list for the list of parametrization cases, since it's homogeneous, unlike the tuples for each case. Applies in several places below, too.

Reply:
Just to confirm, you mean replacing the outermost "tuple of tuples" with a "list of tuples", i.e.
```python
@pytest.mark.parametrize(
    'gate, num_copies, qubits',
    [ # List instead of tuple
        (cirq.SingleQubitGate(), 2, cirq.LineQubit.range(2)),
        (cirq.X ** 0.5, 4, cirq.LineQubit.range(4)),
    ]
)
```